unit/solvers/refinement/string_refinement/union_find_replace.cpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************\
+
+ Module: Unit tests for union_find_replacet in
+   solvers/refinement/string_refinement.cpp
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+
+#include <util/arith_tools.h>
+#include <util/c_types.h>
+#include <util/std_types.h>
+#include <util/std_expr.h>
+#include <solvers/refinement/string_refinement.h>
+
+SCENARIO("union_find_replacea",
```

Comment:
This test could do with some structure to see what is setup code and what is actually being tested.